File: business/src/test/java/no/nav/sbl/dialogarena/soknadinnsending/business/db/TestSupport.java
package no.nav.sbl.dialogarena.soknadinnsending.business.db;

import org.springframework.jdbc.core.support.JdbcDaoSupport;

import javax.inject.Inject;
import javax.sql.DataSource;


public class TestSupport extends JdbcDaoSupport implements RepositoryTestSupport {

    @Inject
    public TestSupport(DataSource dataSource) {
        super.setDataSource(dataSource);
    }
}

File: src/test/kotlin/no/nav/sosialhjelp/soknad/config/TestSupport.kt
package no.nav.sosialhjelp.soknad.config

import org.springframework.jdbc.core.JdbcTemplate
import org.springframework.jdbc.core.support.JdbcDaoSupport
import javax.sql.DataSource

interface RepositoryTestSupport {
    fun getJdbcTemplate(): JdbcTemplate
}

class TestSupport(dataSource: DataSource) : JdbcDaoSupport(), RepositoryTestSupport {
    init {
        super.setDataSource(dataSource)
    }
}